src/pnpq/devices/odl_thorlabs_kbd101.py

```diff
@@ -40,6 +40,14 @@ def __init__(
         self,
         serial_port: str | None = None,
         serial_number: str | None = None,
+        name="Thorlabs",
+        model="KBD101 driver DDS100/M Stage",
+        min_move: int | None = None,
+        max_move: int | None = None,
+        resolution: int | None = None,
+        home_timeout: int | None = None,
+        move_timeout: int | None = None,
+        auto_update: bool | None = None,
```

Comment:
this should be not an optional value because we can specify it False as default like 
```py
auto_update: bool = False
```